third_party/maya/lib/usdMaya/primWriterRegistry.h

```diff
@@ -67,9 +67,9 @@ PXR_NAMESPACE_OPEN_SCOPE
 struct PxrUsdMayaPrimWriterRegistry
 {
     typedef std::function< MayaPrimWriterPtr (
-            MDagPath &curDag,
-            UsdStageRefPtr& stage,
-            const JobExportArgs& args) > WriterFactoryFn;
+            const MDagPath&,
+            const SdfPath&, bool,
+            usdWriteJobCtx&) > WriterFactoryFn;
```

Comment:
The example code on Register (line 81-85) here should be updated to have the same signature as a WriterFactoryFn